src/dealWizard/validation.ts

```diff
@@ -0,0 +1,26 @@
+import { ControllerValidateResult, Rule, ValidationController, ValidationRules } from "aurelia-validation";
+import { IProposalLead } from "../entities/Deal";
+
+export const proposalLeadValidationRules = ValidationRules
+  .ensure<IProposalLead, string>(proposalLead => proposalLead.address)
+  .satisfiesRule("isETHAddress")
```

Comment:
Wdyt about creating an enum or const for `isETHAddress`?
Can then have some kind of map for all the custom validation rules